crates/wit-parser/src/resolve.rs

```diff
@@ -144,95 +180,118 @@ impl Resolve {
                 )
             })
         } else {
-            let id = self.push_file(path)?;
-            Ok((id, vec![path.to_path_buf()]))
+            let ids = self.push_file(path)?;
+            let mut pkg_ids = Vec::new();
+            let mut path_bufs = Vec::new();
+            for id in ids {
+                pkg_ids.push(id);
+                path_bufs.push(path.to_path_buf());
+            }
+            Ok((pkg_ids, path_bufs))
```

Comment:
I think this might be able to stay roughly as it was before, except renaming `id` to `ids`? There's no need to ensure that the two vectors have the same length, it's just that the second vector is the set of files to produce the first.

Reply:
Done.